### ISSUE TITLE
no matching manifest for linux/arm64/v8 in the manifest list entries

### ISSUE THREAD

**Issue by Aopenz** [Josh-XT/Agent-LLM#63]
no matching manifest for linux/arm64/v8 in the manifest list entries

**Comment by Josh-XT** [comment]
Can you be more descriptive about what your machine is running and when you're getting this error?

**Comment by Aopenz** [comment]
This problem has been solved. But I test, this system can no longer run properly on MAC M2, the front section and back end are normal after running open https://127.0.0.1:3000 or other addresses, keep showing network errors, can not load out the correct page.

**Comment by Josh-XT** [comment]
> This problem has been solved. But I test, this system can no longer run properly on MAC M2, the front section and back end are normal after running open https://127.0.0.1:3000 or other addresses, keep showing network errors, can not load out the correct page.

It is not at HTTPS, it is http://localhost

**Comment by Josh-XT** [comment]
Please run:

`docker-compose ps`

To see if your containers are running.

**Comment by Aopenz** [comment]
NAME                   IMAGE                                       COMMAND                  SERVICE             CREATED              STATUS              PORTS
agent-llm-backend-1    ghcr.io/josh-xt/agent-llm-backend:v1.0.6    "uvicorn app:app --h…"   backend             About a minute ago   Up About a minute   5000/tcp, 0.0.0.0:5001->5001/tcp
agent-llm-frontend-1   ghcr.io/josh-xt/agent-llm-frontend:v1.0.6   "docker-entrypoint.s…"   frontend            About a minute ago   Up About a minute   0.0.0.0:80->3000/tcp

**Comment by Aopenz** [comment]
when i use the port 5000:[+] Running 0/0
 ⠋ Container agent-llm-backend-1  Start...                                 0.0s 
Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:5000 -> 0.0.0.0:0: listen tcp 0.0.0.0:5000: bind: address already in use

**Comment by Josh-XT** [comment]
Are you using the docker-compose for mac?

**Comment by Josh-XT** [comment]
> when i use the port 5000:[+] Running 0/0 ⠋ Container agent-llm-backend-1 Start... 0.0s Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:5000 -> 0.0.0.0:0: listen tcp 0.0.0.0:5000: bind: address already in use

Sorry missed earlier you said this was Mac.  I just merged something that should fix that.  You'll need to do this first:

```
docker-compose down
docker-compose -f docker-compose-mac.yml down
```

Then do:

```
docker-compose -f docker-compose-mac.yml up --build -d
```

**Comment by Aopenz** [comment]
=> ERROR [base  7/12] RUN pip install --no-cache-dir -r requirements-mac  1.7s
------                                                                          
 > [base  7/12] RUN pip install --no-cache-dir -r requirements-mac.txt:         
#0 0.269 ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements-mac.txt'
------
failed to solve: executor failed running [/bin/sh -c pip install --no-cache-dir -r requirements-mac.txt]: exit code: 1
